pages/jobs.js

```diff
@@ -34,28 +34,34 @@ export default withData(
-        <P>Bitte beantworte in der Bewerbung – neben dem, was du uns sonst noch sagen möchtest – folgende Fragen: </P>
+        <P>Wir bieten</P>
+        <Editorial.UL>
+          <LI>Mitverantwortung für die Organisation, Gestaltung und Weiterentwicklung des Kulturteils der Republik;</LI>
+          <LI>eine Arbeit im Dialog mit der Republik-Leserschaft auf der Republik-Plattform, auf den sozialen Kanälen und bei Veranstaltungen;</LI>
+          <LI>ein vielfältiges journalistisches Tätigkeitsfeld inklusive redaktionellen Arbeitens;</LI>
+          <LI>die Möglichkeit zu selbstständiger Recherche und Produktion eigener Artikel und Reportagen;</LI>
+          <LI>eine Plattform zum Verfassen von Rezensionen;</LI>
+          <LI>ein abwechslungsreiches, herausforderndes und inspirierendes Umfeld mit agilen Arbeitsweisen.</LI>
+        </Editorial.UL>
 
+        <P>Wir freuen uns auf</P>
         <Editorial.UL>
-          <LI> Was denkst du über unser Produkt? Was gefällt dir daran? Was würdest du ändern?</LI>
-          <LI>Was interessiert dich am Journalismus?</LI>
-          <LI>Was macht dich zu einer guten Coderin, einem guten Coder?</LI>
-          <LI>Hast du Projekte, auf die du besonders stolz bist? Dann freuen wir uns über Links zu Open Source Repos, Websites etc. und eine Begründung, warum du besonders stolz darauf bist.</LI>
+          <LI>eine Kollegin, einen Kollegen, die/der Erfahrung im Kulturjournalismus und  ausgezeichnete Kenntnisse in mindestens einem der klassischen Feuilletongebiete inklusive Pop-Art und Subkulturen mitbringt;</LI>
+          <LI>einen offenen Geist, der sich auch auf unkonventionelle Arbeitsmethoden einlässt und in einem heterogenen Team seine Linie behält;</LI>
+          <LI>eine Journalistin, einen Journalisten, die/der neue Herausforderungen sucht, ein Spezialgebiet hat und bereit ist, dieses weiter zu erforschen;</LI>
+          <LI>eine Persönlichkeit mit eigenständiger Denkweise;</LI>
+          <LI>eine Bewerberin, einen Bewerber, die/der gern redaktionell tätig sein will, Ideen und Konzepte generiert, Texte redigiert und Themengebiete betreut;</LI>
+          <LI>eine Autorin, einen Autoren, die/der sich durch herausragende feuilletonistische Beiträge bereits einen Namen gemacht hat.</LI>
         </Editorial.UL>
 
-        <P>Zum Bewerbungsprozess: Entweder wir laden dich für ein erstes Gespräch ein oder du bekommst leider eine Absage. Falls wir uns nach dem ersten Gespräch eine Zusammenarbeit vorstellen können, laden wir dich für einen Probearbeitstag ein. Wir arbeiten zusammen an einem Feature und du lernst den Rest des Teams kennen. Du und wir haben so eine möglichst gute Grundlage für den finalen Entscheid. Beginn der Arbeit: so bald wie möglich.</P>
-        <P>Wir können es kaum erwarten, dich in unserem Team zu begrüssen!</P>
+        <P>Interessiert? Neugierig? Senden Sie Ihre vollständigen Bewerbungsunterlagen an <a href='mailto:personal@republik.ch'>personal@republik.ch</a>.</P>
```

Comment:
This should be a green A element from the styleguide. Looks broken now:
<img width="937" alt="screen shot 2018-05-16 at 16 29 20" src="https://user-images.githubusercontent.com/410211/40123369-613ab270-5926-11e8-82cd-9d44e488ba11.png">